pymongo/client_session.py

```diff
@@ -1000,6 +1009,24 @@ def __copy__(self):
         raise TypeError("A ClientSession cannot be copied, create a new session instead")
 
 
+class _EmptyServerSession(object):
+    def __init__(self, generation):
+        self.generation = generation
+        self.last_use = time.monotonic()
+        self.lsid = None
+        self.dirty = False
+        self.started = 0
+
+    def timed_out(self, session_timeout_minutes):
```

Comment:
timed_out is not needed is it?

Reply:
It is needed because of `session_pool.return_server_session()`.

Reply:
I think it would be cleaner for the session_pool to have no knowledge of the _EmptyServerSession. Let's avoid calling session_pool.return_server_session or session_pool.return_server_session_no_lock with a _EmptyServerSession.